airtable.go

```diff
@@ -59,6 +58,11 @@ func (t *GenericTable) List(opts Options) ([]Record, error) {
 		q.Add("maxRecords", fmt.Sprint(opts.MaxRecords))
 	}
 
+	// add offset
+	if opts.Offset != "" {
+		q.Add("offset", fmt.Sprint(opts.Offset))
```

Comment:
Since `opts.Offset` is a string, the `fmt.Sprint` is unnecessary. But, if `Offset` is indeed an int and you want to "cast" it into a string, what you have here is valid